parts/kubelet-restart.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+if [ ! -e /opt/azure/containers/runcmd.complete ]; then
+  exit 0
+fi
+
+ready=$(kubectl --kubeconfig /var/lib/kubelet/kubeconfig get nodes/$(hostname) -o jsonpath='{.status.conditions[?(@.type=="Ready")].status}')
```

Comment:
Not sure I understand the check here...the kubelet is seeing if it shows up as "Ready" via `kubectl`?